components/prism-lawtex.js

```diff
@@ -0,0 +1,32 @@
+Prism.languages.lawtex = {
+  'comment': [
+    {
+      pattern: /(^|[^\\])\/\*[\s\S]*?(?:\*\/|$)/,
+      lookbehind: true,
+      greedy: true
+    },
+    {
+      pattern: /(^|[^\\:])\/\/.*/,
+      lookbehind: true,
+      greedy: true
+    }
+  ],
+  'string': {
+    pattern: /(^|[^\\](?:\\\\)*)(")(?:\\[\s\S]|\$\([^)]+\)|\$(?!\()|`[^`]+`|(?!\2)[^\\`$])*\2/,
```

Comment:
```suggestion
    pattern: /(^|[^\\](?:\\\\)*)"(?:\\[\s\S]|\$\([^)]+\)|\$(?!\()|`[^`]+`|[^"\\`$])*"/,
```

Reply:
Sorry I'm new to programming so I'm not sure of all the changes you have suggested. I'll try my best to evaluate and reply to them. 
This does seem to make the expression simpler since the language only allows for double quotes to open a string. Thank you.